Userland/Services/WindowServer/WindowManager.cpp

```diff
@@ -1052,6 +1052,25 @@ Gfx::IntRect WindowManager::desktop_rect() const
     };
 }
 
+Gfx::IntRect WindowManager::arena_rect_for_type(WindowType type) const
```

Comment:
s/arena/area/ (?)

Reply:
But even then I don't get what the name is supposed to mean. "screen_rect_for_window_type" (?)

Reply:
That was not a typo: "Arena" as in "the space where you can move around". I'm not sure whether `screen_rect` is more intuitive than `arena`, but at least it looks less like a typo.